linchpin/README.md

```diff
@@ -124,7 +138,20 @@ It is also possible to set up the scheduling on the *master* of a permanent *tar
 Examples
 --------
 
-[examples](examples)
+[Testing a code change running a single test on temporary target](examples/example1)
```

Comment:
On other places you have `Testing a change...` please make that consistent.

Reply:
Made consistent.